frontend/src/components/payment/cards/CardInfo.tsx

```diff
@@ -0,0 +1,63 @@
+import { useTranslation } from 'next-i18next'
+import { useTheme } from 'next-themes'
+import { FunctionComponent, MouseEventHandler } from 'react'
+import { IMAGE_BASE_URL } from '../../../env'
+import { PaymentCard } from '../../../types/Payment'
+import Image from '../../Image'
+import styles from './CardInfo.module.scss'
+
+interface Props {
+  card: PaymentCard
+  onClick?: MouseEventHandler
+  className?: string
+}
+
+/**
+ * Converts country code to corresponding emoji flag
+ * @param code ISO-3166-1 alpha-2 code of coutnry
+ * @returns emoji flag string (two regional indicator characters)
+ */
+function countryCodeToFlag(code: string): string {
+  // Consistent 127,397 offset from capital to regional indicator
+  return code
```

Comment:
What's this going to return if I call this with a non iso code string?

Reply:
It will return a string of arbitrary unicode characters (most likely emoji).

I considered defensively coding this function, but outside of introducing a lookup for all possible iso codes (which seems excessive) I'm not sure there's much to be done.

The only strings currently being fed in come from the backend which is specified to return ISO codes only.

Reply:
https://github.com/danalloway/react-country-flag

Somebody else did the work? :D